JS/models/militaryTypes.js

```diff
@@ -0,0 +1,7 @@
+const militaryTypes = {
```

Comment:
The same as for comment regarding Experimantal Types.

Reply:
Done